permissions/resource_rule_set_test.go

```diff
@@ -0,0 +1,347 @@
+package permissions
+
+import (
+	"fmt"
+	"net/url"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/iam"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/databricks/terraform-provider-databricks/qa"
+)
+
+var (
+	TEST_ACCOUNT_ID                 = "cb376b18-60fa-4058-b2cd-dd85acf63165"
```

Comment:
in golang, we don't use CONSTANT_CASE. we use `camelCase`. please don't trigger our linters.